std/experimental/typecons.d

```diff
@@ -271,15 +271,15 @@ if (Targets.length >= 1 && allSatisfy!(isInterface, Targets))
                 static if (is(Source == class) || is(Source == interface))
                 {
                     // BUG: making private should work with NVI.
-                    protected final inout(Object) _wrap_getSource() inout @safe
```

Comment:
No, because the declaration of the class is `final class Impl : Structural!StructuralType, Targets`. If the class is final then all its methods are implicitly final, i.e you can't create a subclass that overrides `_wrap_getSource()`